dbfit-java/informix/src/test/java/dbfit/environment/BigDecimalExactClassTransformerTest.java

```diff
@@ -0,0 +1,50 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+
+import java.math.BigDecimal;
+import java.lang.Double;
```

Comment:
No need of explicit import of `Double` - `java.lang.*` is automatically imported.